index.html

```diff
@@ -204,44 +204,47 @@ <h2 id="media-text">
         </div>
         <div class="languages">
             <h3 id="lang">
-                <p>Languages</p>
-                <img src="images\Union.svg">
+                <p role="article">Languages</p>

```

Comment:
:red_circle: please the Languages is a subheading and should be in an h tag, not a p tag. kindly fix that